packages/quantic/force-app/examples/main/lwc/exampleQuanticBreadcrumbManager/exampleQuanticBreadcrumbManager.js

```diff
@@ -1,27 +1,37 @@
-import { api, LightningElement, track } from 'lwc';
+import {api, LightningElement, track} from 'lwc';
 
 export default class ExampleQuanticFacetManager extends LightningElement {
   @api engineId = 'quantic-breadcrumb-manager-engine';
   @track config = {};
   isConfigured = false;
 
   pageTitle = 'Quantic Breadcrumb Manager';
-  pageDescription = 'The QuanticBreadcrumbManager component creates breadcrumbs that display a summary of the currently active facet values.';
-  
+  pageDescription =
+    'The QuanticBreadcrumbManager component creates breadcrumbs that display a summary of the currently active facet values.';
+
   options = [
-      {
-        attribute: 'categoryDivider',
-        label: 'Category Divider',
-        description: 'A character that divides each path segment in a category facet breadcrumb.',
-        defaultValue: '/'
-      },
-      {
-        attribute: 'collapseThreshold',
-        label: 'Collapse Threshold',
-        description: 'Maximum number of displayed breadcrumb values. When more values are selected, additional values appear under the "More" button.',
-        defaultValue: 5
-      },
-  ]; 
+    {
+      attribute: 'categoryDivider',
+      label: 'Category Divider',
+      description:
+        'A character that divides each path segment in a category facet breadcrumb.',
+      defaultValue: '/',
+    },
+    {
+      attribute: 'collapseThreshold',
+      label: 'Collapse Threshold',
+      description:
+        'Maximum number of displayed breadcrumb values. When more values are selected, additional values appear under the "More" button.',
+      defaultValue: 5,
+    },
+    {
+      attribute: 'isSearch',
```

Comment:
I think using some sort of option like this is a good idea. However I'm concerned that `isSearch` won't scale if/when we end up with more than two use cases.

How about using something like a `useCase` option which accepts a string and can be set to either `"search"`, `"insight"`, `"recommendation"`, etc...

Reply:
good catch, I was kind of hesitated between string and boolean for the same, now I can be more sure about string